index.html

```diff
@@ -26,23 +26,25 @@ <h1>Where in the world?</h1>
     </nav>
 
     <header class="container filter-content margin-y">
-        <form>
-            <input type="text" placeholder="Ingresa País">
+        <form class="form-search" id="form">
+            <i class="fa-solid fa-magnifying-glass"></i>
+            <input type="text" placeholder="Enter Country" id="inputForm">
         </form>
-        <div>
+        <div class="custom-select" style="width:200px;">
```

Comment:
You rather want to add the `width: 200px` in the `custom-select` class instead.
There is a thing called `Specificity` in `CSS`, which is the `weight` of a CSS selector. I will explain it in more detail in today's meeting.
Docs: https://developer.mozilla.org/en-US/docs/Web/CSS/Specificity